src/five/localsitemanager/localsitemanager.txt

```diff
@@ -40,11 +37,11 @@ setting up a test utility.
 
     >>> class ITestUtility(interface.Interface): pass
     >>> class TestUtility(object):
-    ...     interface.implements(ITestUtility)
     ...     def __init__(self, id):
     ...         self.id = id
     ...     def __repr__(self):
     ...         return '<Utility %s "%s">' % (self.__class__.__name__, self.id)
+    >>> TestUtility = interface.implementer(ITestUtility)(TestUtility)
```

Comment:
You're right, this was my mistake because I had used wrong doctest syntax at first and thought it would not work at all. I had used...

```python
  >>> @decorator...
  >>> class Foo...
```

instead of the correct...

```python
  >>> @decorator...
  ... class Foo...
```

e6e1aa1105267e2e2594906f834f8d1df619e1ca fixes it.